src/stories/layouts/Wizard/Wizard.js

```diff
@@ -29,6 +31,7 @@ export const Wizard = ({ children, labels }) => {
   // update step value
   const onStepChange = (stats) => {
     setStep(WizInstance.currentStep);
+    stepHandler(WizInstance.currentStep);
```

Comment:
This seems redundant as `stepHandler` and `setStep` do basically the same thing but in parent and child component. I tried to remove `setStep` and use stepHandler and its step value but Next button disables as it doesn't fetch the latest step.

Reply:
or maybe we can just remove the `step` altogether and yes just `WizInstance.currentStep` wherever needed; for example, while passing it in the Controls component.
`{
          WizInstance && <Controls currentStep={step -> WizInstance.currentStep}
            WizInstance={WizInstance}
            experimentId={experimentId}
            showNext={showNext} />
        }`

Reply:
I think the difficulty comes from having to know what the is going on with the Wizard but in the Experiment component. So `stepHandler` helps to tell Experiment what step the Wizard is in. `setStep` is only used by the Wizard to update the step. So, not sure if can live with only one of the two.

If there is a way to access WizInstance's functionality in Experiment then I think we could try what you suggest in your second comment. Do you know how to access that from within Experiment?

Reply:
So I replaced the `currentStep` attribute of `<Control>` with `WizInstance.currentStep` and it works perfectly fine
<img width="618" alt="Screen Shot 2021-04-08 at 1 53 44 PM" src="https://user-images.githubusercontent.com/23450260/114095445-466a5700-9872-11eb-8511-863af9f5e4a1.png">

Reply:
Also, 

> I think the difficulty comes from having to know what step the Wizard is in but in the Experiment component. So `stepHandler` helps to tell Experiment what step the Wizard is in. `setStep` is only used by the Wizard to move back and forth between steps. So, not sure if we can live with only one of the two.
> 
> If there is a way to access WizInstance's functionality in Experiment then I think we could try what you suggest in your second comment. Do you know how to access that from within Experiment?

`stepHandler` stays because it is used in experiment.js but the one we can remove is the local `step` state for Wizard.js
<img width="618" alt="Screen Shot 2021-04-08 at 1 58 52 PM" src="https://user-images.githubusercontent.com/23450260/114095672-8e897980-9872-11eb-8dab-ecc27b955b40.png">

Reply:
It's not a big deal, it is just one more state haha

Reply:
Gotcha, that makes sense 👍. I didn't write the Wizard component and it doesn't have to do with what this PR is trying to accomplish so I think we can remove the `step` variable in a separate PR to keep things well contained?